docs/performance/graph-optimizations.md

```diff
@@ -56,20 +57,26 @@ These optimizations include complex node fusions. They are run after graph parti
-| BERT Embedding Layer Fusion     | cpu or cuda        | Fuse BERT embedding layer, layer normalization and attention mask length    |
-| Attention Fusion                | cpu or cuda        | Attention mask has approximation in cuda execution provider                 |
-| Skip Layer Normalization Fusion | cpu or cuda        | Fuse bias of fully connected layer, skip connection and layer normalization |
-| Bias GELU Fusion                | cpu or cuda        | Fuse bias of fully connected layer and GELU activation                      |
-| GELU Approximation              | cuda               | Erf is approximated by a formula using tanh function                        |
+| GEMM Activation Fusion          | CPU                |                                                                             |
+| Matmul Add Fusion               | CPU                |                                                                             |
+| Conv Activation Fusion          | CPU                |                                                                             |
+| GELU Fusion                     | CPU or CUDA        |                                                                             |
+| Layer Normalization Fusion      | CPU or CUDA        |                                                                             |
+| BERT Embedding Layer Fusion     | CPU or CUDA        | Fuse BERT embedding layer, layer normalization and attention mask length    |
+| Attention Fusion*               | CPU or CUDA        |                                                                             |
+| Skip Layer Normalization Fusion | CPU or CUDA        | Fuse bias of fully connected layer, skip connection and layer normalization |
+| Bias GELU Fusion                | CPU or CUDA        | Fuse bias of fully connected layer and GELU activation                      |
+| GELU Approximation*             | CUDA               | Disabled by default. Enable with [kOrtSessionOptionsEnableGeluApproximation](https://cs.github.com/microsoft/onnxruntime/blob/175acf08f470db0bb2e4b8eefe55cdeb87c8b132/include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h?q=kOrtSessionOptionsEnableGeluApproximation#L52) |
+
+
+<details>
+  <summary>
+    Approximations (click to expand)
+  </summary>
 
-To optimize inference performance of BERT model, approximation is used in GELU approximation and Attention fusion for cuda execution provider. There might be slight difference in result. The impact on accuracy could be neglected based on our evaluation: F1 score for a BERT model on SQuAD v1.1 is almost same (87.05 vs 87.03).
+  To optimize performance of <a href="https://en.wikipedia.org/wiki/BERT_(language_model)">BERT</a>, approximation is used in GELU Approximation and Attention Fusion for CUDA execution provider. The impact on accuracy is negligible based on our evaluation: F1 score for a BERT model on SQuAD v1.1 is almost same (87.05 vs 87.03).
```

Comment:
No I actually was messing around with this because I thought maybe the `<details>` tag disabled markdown entirely and switched to HTML mode, but the markdown style link still works. Done.